rotkehlchen/db/upgrades/v40_v41.py

```diff
@@ -22,15 +36,41 @@ def _add_cache_table(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _add_cache_table')
 
 
+def _get_unrelated_mappings(read_cursor: 'DBCursor') -> dict:
+    """Get the unrelated key-value pairs from `settings`"""
+    log.debug('Enter _get_unrelated_mappings')
+    settings = read_cursor.execute('SELECT name, value FROM settings').fetchall()
+    log.debug('Exit _get_unrelated_mappings')
+    return {name: value for name, value in settings if name in SETTINGS_MOVED}
```

Comment:
don't create a function for this is just a query that you can do in the other function